webapp/src/libs/services/DocumentDeleteService.ts

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+import { BaseService } from './BaseService';
+
+export class DocumentDeleteService extends BaseService {
+    public deleteDocumentAsync = async (
```

Comment:
I suggest moving this to `DocumentImportService.ts` to keep all the document functionality together.